confd/templates/janitor.properties.tmpl

```diff
@@ -1,2 +1,191 @@
-# Disable Janitor Monkey
-simianarmy.janitor.enabled = false
+# see documentation at:
+# https://github.com/Netflix/SimianArmy/wiki/Configuration
+
+# By default Janitor Monkey wakes up every hour
+simianarmy.scheduler.frequency = {{getv "/simianarmy/scheduler/frequency" "1"}}
```

Comment:
This is actually one of the problems I have with the key-value scheme: one `/simianarmy/scheduler` here again would mean that both Chaos and Janitor Monkey always do things at the same time.